code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -495,7 +495,7 @@ datum
 				..()
 
 		fooddrink/alcoholic/bojack // Bar Contest Winner's Drink
-			name = "Bo Jack Daniel's"
+			name = "bo jack daniel's"
```

Comment:
i think this should be capitalized because it's a brand

Reply:
Fair. Adding